source/ast/expressions/AssertionExpr.cpp

```diff
@@ -1186,6 +1541,14 @@ AssertionExpr& DisableIffAssertionExpr::fromSyntax(const DisableIffSyntax& synta
 
     checkSampledValueExpr(cond, context, false, diag::DisableIffLocalVar, diag::DisableIffMatched);
 
+    // If condition of `disable iff` can be evaluated only as `true` (`1`) than sequence property
+    // is no match
+    if (const auto evaluatedValue = context.tryEval(cond);
+        evaluatedValue.isInteger() && evaluatedValue.integer().countOnes()) {
```

Comment:
`isTrue`

Reply:
fixed for all locations when I used `countOnes`